src/coinqvest/inc/admin/js/coinqvest-admin-ajax-handler.js

```diff
@@ -8,27 +8,23 @@ jQuery( document ).ready( function( $ ) {
 
         event.preventDefault();
 
-        var ajax_form_data = $("#coinqvest_ajax_form").serialize();
-
-        ajax_form_data = ajax_form_data+'&ajaxrequest=true&submit=Submit+Form';
-
         $.ajax({
             url: params.ajaxurl,
             dataType: 'json',
             type: 'post',
-            data: ajax_form_data,
+            data: $("#coinqvest_ajax_form").serialize() + '&ajaxrequest=true&submit=Submit+Form',
             success: function (response) {
 
                 $("#coinqvest_form_feedback").removeClass().addClass(response.success === true ? "notice notice-success" : "notice notice-error").html(response.message).fadeIn();
 
-                if(response.redirect) {
-                    window.location = response.redirect;
-                }
-
                 if (response.success === true && response.clear === true) {
                     $('form#coinqvest_ajax_form')[0].reset();
                 }
 
+                if(response.redirect) {
+                    window.location = response.redirect;
+                }
+
```

Comment:
I changed the order here because I think code after window.location changes isn't executed anymore.